src/services/instance/items/items.test.js

```diff
@@ -18,13 +18,13 @@ const TEST_ITEM_JSON = JSON.parse(
 
 test.beforeEach(authContext);
 
-test("fetchItems:200", async t => {
+test.serial("fetchItems:200", async t => {
```

Comment:
comment in the README explaining why these are running serially. Include all of the other tests that are also now serially run.

Reply:
also include the extra variables you introduced under the `Development` section